README.md

```diff
@@ -57,4 +57,27 @@ If you are on call or asked to fix an immediate issue, check out our doc on
 [how support works](/playbooks/support.md#readme) and reference our
 [support wiki](https://github.com/artsy/potential/wiki) 🔒 for up-to-date playbooks on how to solve issues.
 
-<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
+<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
+
+## About Artsy
+
+<a href="https://www.artsy.net/">
+  <img align="left" src="https://avatars2.githubusercontent.com/u/546231?s=200&v=4"/>
+</a>
+
+This project is supported by [Artsy][footer_website], a marketplace for buying
```

Comment:
couple thoughts that you are welcome to take or leave: 
- `supported by` feels a little flat - what about something like "This project is the work of/is primarily maintained by/was created by engineers at Artsy"?
- `a marketplace for buying and selling art online` - I think this could be more consistent with how we discuss Artsy elsewhere, and I would be careful about "selling" since that makes it sound like anyone can sell on Artsy. Some language stolen from Simon's recent email on discussing Artsy:
```
- Artsy is the world’s leading and largest online art marketplace and platform for discovering art
- We connect buyers and sellers across 190 countries spanning six continents, with an average transaction distance of 3000 miles
- We now partner with 3000+ galleries, institutions and auction houses across the globe
```

Reply:
Great stuff, thanks!!